tests/utils/test_audio_utils.py

```diff
@@ -271,6 +324,58 @@ def test_spectrogram_integration_test(self):
         self.assertEqual(spec.shape, (257, 732))
         self.assertTrue(np.allclose(spec[:64, 400], expected))
 
+        mel_filters = mel_filter_bank(
+            num_frequency_bins=256,
+            num_mel_filters=400,
+            min_frequency=20,
+            max_frequency=8000,
+            sampling_rate=16000,
+            norm=None,
+            mel_scale="kaldi",
+            use_torchaudio_version=True,
+        )
+
+        mel_filters = np.pad(mel_filters, ((0, 1), (0, 0)))
+
+        spec = spectrogram(
+            waveform,
+            window_function(400, "povey", periodic=False),
+            frame_length=400,
+            hop_length=160,
+            fft_length=512,
+            power=2.0,
+            center=False,
+            pad_mode="reflect",
+            onesided=True,
+            preemphasis=0.97,
+            mel_filters=mel_filters,
+            log_mel="log",
+            mel_floor=1.1920928955078125e-07,
+            remove_dc_offset=True,
```

Comment:
This is a snippet of how you can get equivalence with `kaldi.fbank` (with defaults arguments) using `audio_utils`.

Reply:
Cool! One thing: it doesn't use the `use_torchaudio_version` argument? Is this only strictly used for `torchaudio.stft`? Not `torchaudio.kaldi.fbank`?

Reply:
I actually used it in the `mel_filters` (L335) !